packages/tests/test-utils/create-mock-menu-actions.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+
+export function createMockMenuAction( { onClick = () => null }: { onClick?: () => void } = {} ) {
```

Comment:
Add options to overwrite

Reply:
Add return type, so it will be clear what it retrurns

Reply:
NO